backend/src/test/java/gov/cdc/usds/simplereport/api/model/TestEventExportTest.java

```diff
@@ -194,4 +194,30 @@ void sendCorrectionReason() {
     assertEquals("Cold feet", exportedEvent.getCorrectionReason());
     assertEquals(originalEventId, exportedEvent.getCorrectedResultId());
   }
+
+  @Test
+  void sendPatientInfo() {
+    // GIVEN
+    Organization org = _dataFactory.createValidOrg();
+    Facility facility = _dataFactory.createValidFacility(org);
+    Person person = _dataFactory.createFullPerson(org);
+    TestEvent testEvent =
+        _dataFactory.createTestEvent(person, facility, TestResult.NEGATIVE, false);
+
+    // WHEN
+    TestEventExport exportedEvent = new TestEventExport(testEvent);
+
+    // THEN
+    assertEquals("Fred", exportedEvent.getPatientFirstName());
```

Comment:
Could you also add tests to confirm that this works with a null country and with a non-US country?

Reply:
good call, fixed in https://github.com/CDCgov/prime-simplereport/pull/3392/commits/9e36464892ea2f6f2ed4a8f078998abdc488518a